eip-0001.md

```diff
@@ -15,8 +15,7 @@ Currently, the Ergo node wallet is able to search for boxes protected only by si
 Development would involve scanning the blockchain state independently by the off-chain portion of the dApp itself with handling forks, confirmation numbers, and so on.
 
 This Ergo Improvement Proposal focused on extending the wallet to be able to serve the needs of external applications by providing 
-a flexible scanning interface and the possibility for applications to register scans with the wallet to ensure that they are tracked.
-successfully passed scanning are indeed belong to the application.
+a flexible scanning interface and the possibility for applications to register scans with the wallet to ensure that they are tracked. Scans that have successfully passed are considered to belong to the application.
```

Comment:
Maybe being specific here and saying something like:

`Scans that have successfully passed are considered to belong to the application (meaning that the wallet will no longer track said boxes).`